doc/man1/openssl-smime.pod.in

```diff
@@ -233,9 +235,11 @@ option is present B<CRLF> is used instead.
 
 =item B<-certfile> I<file>
 
-Allows additional certificates to be specified. When signing these will
-be included with the message. When verifying these will be searched for
-the signers certificates.
+Provide additional certificates.
+When signing, these will be included (unless the B<-nocerts> option is given)
+in the message to help chain building and to provide extra signer certificates.
+When verifying, these will be searched first for signer certificates.
```

Comment:
```suggestion
Provide additional certificates.
When signing, these will be included in the message as candidate untrusted CAs to assist in verification.
When verifying, these take precedence over the signer certificates in the message as a source of candidate signers.
With B<-nointern> these will be the only signer certificates considered.
Note that a message can have multiple signers.
When verifying a multi-signed message, all the signatures must be valid, and so all the signer certificates need to be available.
```

Reply:
Basically taken over as above, but also here we must take into account than subtle difference with PKCS#7 mentioned above, so kept the `(unless the B<-nocerts> option is given)` part.